### PR TITLE
Submission for Wesley Meredith

### PR DESCRIPTION
**Pull request by wesmered**
As the commit descriptions will detail, I mostly had issues with embedding a photo and properly adding a space after the hyphen for lists. 

Also, in the process of making this pull request, I noticed someone else had posted about their issue with embedding a photo, so I need to remember to better monitor the HW pages. This would have saved a little time trying out different ways to appropriately embed the file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
-# hw01
-Repository for homework 01
+# hw01 - Wesley Meredith
+
+![](IMG_0575.jpg)
```
Comment (bensoltoff) [inline README.md:3]: Hey, nice pic!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+## Academic Profile
+For a general overview of my research experiences, check out my [ResearchGate profile](https://www.researchgate.net/profile/Wesley_Meredith).
+
+## Non-Academic Interests
+- Reading, Science Fiction and Nonfiction
+- Bike Riding
+- Cooking and Bread Baking
+- Listening to Podcasts
+
+__Miscellaneous: please use they/them pronouns or my first name for me__
```
Comment (bensoltoff) [inline README.md:22]: I will try to remember this in the future

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**          |    x       |              |            |

Remarks:

* Lots of Markdown elements used
* No extraneous files in the repo

**Comment by mgneely** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |X           |              |            |
| **Coding strategy**         |X           |              |            |
| **Presentation**            |X           |              |            |
| **Achievement, creativity** |X           |              |            |
| **Ease of access**          |X           |              |            |

Remarks:

* Met requirement of using 4 Markdown Elements
* Went above and beyond with markdown elements, especially with use of headers and bolding text!
* Nice use of URL!

**Pull request closed without merging**